mods/default/functions.lua

```diff
@@ -754,12 +754,12 @@ function default.set_inventory_action_loggers(def, name)
 	end
 	local on_put = def.on_metadata_inventory_put or nop
 	def.on_metadata_inventory_put = function(pos, listname, index, stack, player)
-		default.log_player_action(player, "moves", stack:get_name(), "to", name, "at", pos)
+		default.log_player_action(player, "moves", stack:to_string(), "to", name, "at", pos)
 		return on_put(pos, listname, index, stack, player)
 	end
 	local on_take = def.on_metadata_inventory_take or nop
 	def.on_metadata_inventory_take = function(pos, listname, index, stack, player)
-		default.log_player_action(player, "takes", stack:get_name(), "from", name, "at", pos)
+		default.log_player_action(player, "takes", stack:to_string(), "from", name, "at", pos)
```

Comment:
I believe this was intentional as to not print potentially huge amounts of metadata to the log

Reply:
what I care about is name and amount. but I can see what you're talking about

Reply:
Then why not format just name and amount?

Reply:
> Then why not format just name and amount?

that would mean storing default:cobble 1 to the chest instead of storing default:cobble to the chest. Would that be okay?

Reply:
do you mean renaming name_and_amount to name_and_count or deleting that function completely?